### ISSUE TITLE
Run travis on cron

### ISSUE THREAD

**Issue by springmeyer** [mapbox/gdal-tiling-bench#28]
I think we should run this repo on a cron to ensure its always green when we come back to utilize. I propose running on a monthly interval. Sound good?  /cc @mapsam @BergWerkGIS @dnomadb

**Comment by mapsam** [comment]
@springmeyer definitely. Let's use the TravisCI cron feature (same as mapnik docs). I'll look into this come 2017!
